src/pages/es/core-concepts/routing.md

```diff
@@ -337,3 +337,26 @@ export async function getStaticPaths({paginate}) {
 const { page } = Astro.props;
 const params = Astro.params;
 ```
+
+
+## Excluyendo páginas
+
+Puedes excluir páginas, o incluso directorios completos de ser generados añadiendo el prefijo (`_`).
+
+Esto permite crear páginas privadas, y también co ubicar *tests*, utilidades y componentes con su páginas relacionadas, evitando generar archivos `.html` en el directorio `dist/`.
```

Comment:
I was not able to find a definition for "co ubicar" in Spanish, I think just "colocar" fits better in this paragraph.

```suggestion
Esto permite crear páginas privadas, y también colocar *tests*, utilidades y componentes con sus páginas relacionadas, evitando generar archivos `.html` en el directorio `dist/`.
```

Reply:
@agustinmulet

Reply:
```suggestion
Esto te permite crear páginas privadas, y también incluir otros tipos de archivos como *tests*, utilidades y componentes junto con las páginas a donde pertenecen, evitando generar archivos `.html` en el directorio `dist/`.
```